code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -46,13 +46,14 @@ emp_act
 		var/armor = getarmor_organ(organ, "bullet")
 		if((P.embed && prob(20 + max(P.damage - armor, -10))))
 			var/obj/item/weapon/shard/shrapnel/SP = new()
-			(SP.name) = "[P.name] shrapnel"
+			SP.name = "[P.name] shrapnel"
 			if(P.ammo_casing && P.ammo_casing.caliber)
-				(SP.desc) = "[SP.desc] It looks like it is a [P.ammo_casing.caliber] caliber round."
+				SP.desc = "[SP.desc] It looks like it is a [P.ammo_casing.caliber] caliber round."
 			else
-				(SP.desc) = "[SP.desc] The round's caliber is unidentifiable."
-			(SP.loc) = organ
-			organ.embed(SP)
+				SP.desc = "[SP.desc] The round's caliber is unidentifiable."
+			throw_alert("embeddedobject", /obj/screen/alert/embeddedobject)
+			organ.embedded_objects |= SP
+			SP.forceMove(src)

```

Comment:
Yeah, more than likely you can't pull out a bullet anyways. Anything else should be fine to yank out